src/yunohost/utils/ldap.py

```diff
@@ -31,12 +31,16 @@ def _get_ldap_interface():
     global _ldap_interface
 
     if _ldap_interface is None:
-        # Instantiate LDAP Authenticator
-        AUTH_IDENTIFIER = ('ldap', 'as-root')
-        AUTH_PARAMETERS = {'uri': 'ldapi://%2Fvar%2Frun%2Fslapd%2Fldapi',
-                           'base_dn': 'dc=yunohost,dc=org',
-                           'user_rdn': 'gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth'}
-        _ldap_interface = init_authenticator(AUTH_IDENTIFIER, AUTH_PARAMETERS)
+
+        conf = { "vendor": "ldap",
+                 "name": "as-root",
+                 "parameters": { 'uri': 'ldapi://%2Fvar%2Frun%2Fslapd%2Fldapi',
+                                 'base_dn': 'dc=yunohost,dc=org',
+                                 'user_rdn': 'gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth' },
```

Comment:
```suggestion
                                 'user_rdn': 'gidNumber=0,uidNumber=0,cn=peercred,cn=external,cn=auth' },
```

I know it's the same as the original text but just checking - the `+` is not a typo, right!?

Reply:
Eeeeh no idea :s (@Josue-T do you know about this ?)

Reply:
Well, `+` is not same than `,`. `,` is the separation between each element of the LDAP tree. For `+` it's different, it's really specific, it's liked to the root authentication. Don't really know exactly what it mean but probably something like gidNumber=0 and uidNumber=0. Anyway `+` true.